pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/SegmentPushUtils.java

```diff
@@ -264,6 +264,10 @@ public static void sendSegmentUriAndMetadata(SegmentGenerationJobSpec spec, Pino
               headers.add(new BasicHeader(FileUploadDownloadClient.CustomHeaders.DOWNLOAD_URI, segmentUriPath));
               headers.add(new BasicHeader(FileUploadDownloadClient.CustomHeaders.UPLOAD_TYPE,
                   FileUploadDownloadClient.FileUploadType.METADATA.toString()));
+              if (spec.getPushJobSpec() != null) {
+                headers.add(new BasicHeader(FileUploadDownloadClient.CustomHeaders.COPY_SEGMENT_TO_DEEP_STORE,
+                    String.valueOf(spec.getPushJobSpec().getMoveToDeepStoreForMetadataPush())));
```

Comment:
I guess I was a bit confused with the PushJobSpec definition name "move" and the CustomHeader field "copy". so it actually resembles a fils system "copy" but not a "move" correct? maybe renaming to avoid confusion?

Reply:
good catch! it started off being "move" everywhere because of some other reason, and when I made it "copy" i missed updating it in PushJobSpec. Fixed